modules/contact/test/tests/mechanical-small-problem/gold/frictionless-nodal-lm-mortar-disp-action_out.e

```diff
@@ -0,0 +1 @@
+frictionless-nodal-lm-mortar-disp_out.e
```

Comment:
It looks like the non-contact test was re-golded. Was the reason for that the renaming of the variables? Did you verify that the results didn't actually change?

Reply:
Yes, same as other re-gold: I checked the results visually.